src/main/java/pro/quicksense/common/AirwallexConstant.java

```diff
@@ -0,0 +1,26 @@
+package pro.quicksense.common;
+
+public class AirwallexConstant {
+
+    public static final String API_AIRWALLEX_ROOT = "https://api-demo.airwallex.com/api/v1";
```

Comment:
ditto

Reply:
There are multiple APIs based on this root API, how do I append them if the root API is configured in the `application.yml`?

Reply:
```
@Value("${airwallex.api_airwallex_root}")
public static final String API_AIRWALLEX_ROOT
```
I think we can get value from `@Value` annotation.